src/borg/archive.py

```diff
@@ -569,7 +569,7 @@ def extract_item(self, item, restore_attrs=True, dry_run=False, stdout=False, sp
         path = os.path.join(dest, item.path)
         # Attempt to remove existing files, ignore errors on failure
         try:
-            st = os.lstat(path)
+            st = os.stat(path, follow_symlinks=False)
```

Comment:
I can get behind that.

lchmod on Linux is easy, though; it doesn't make sense, so it doesn't exist (Linux ignores permissions on symlinks, like most Unices do).